### ISSUE TITLE
Question:   Do you have a tool to replay results from the global results.log

### ISSUE THREAD

**Issue by coneheadusa** [hpc/pavilion2#364]
Is there a tool that would allow one to replay a test from the contents of the global results.log file?

**Comment by Paul-Ferrell** [comment]
No, there isn't, and that would be a bit hard to accomplish as the result log doesn't contain all the test details. 

We could fairly easily make tests replayable given the test run directory, which probably still exists anyway.
